third_party/icu.BUILD

```diff
@@ -172,6 +172,56 @@ cc_binary(
     ],
 )
 
+genrule(
+    name = "pkgdata_inc",
+    outs = ["pkgdata.inc"],
+    cmd = select({
+        "@platforms//os:macos": r"""
+echo "
```

Comment:
Maybe move these scripts into something like `PKGDATA_INC_MACOS_CMD = ...`-variables above to make this easier to read? It's not super obvious where one commands ends and the next one starts.